docs/compatibility.md

```diff
@@ -2,262 +2,161 @@ List of supported features, depending on the Docker Compose version.
+|   - resources                |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - limits                 |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - reservations           |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|   - restart_policy           |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - condition: none        |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - condition: on-failure  |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - condition: any         |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - delay                  |  S  |  S  |  S  |  S  |  S  |  S  |  S  |  S  |     |     |     |     |     |
+|     - max_attempts           |  S  |  S  |  S  |  S  |  S  |  S  |  S  |  S  |     |     |     |     |     |
+|     - window                 |  S  |  S  |  S  |  S  |  S  |  S  |  S  |  S  |     |     |     |     |     |
+|   - rollback_config          |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|     - delay                  |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|     - failure_action         |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|     - max_failure_ratio      |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|     - monitor                |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|     - order                  |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|     - parallelism            |  S  |     |     |     |     |     |     |     |     |     |     |     |     |
+|   - update_config            |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - delay                  |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - failure_action         |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - max_failure_ratio      |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - monitor                |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|     - order                  |  SK |  SK |  SK |  SK |     |     |     |     |     |     |     |     |     |
+|     - parallelism            |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+| - devices                    |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - depends_on                 |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
```

Comment:
Doesn't work on `swarm`, there is no notion of `depends_on` in swarm, the key is ignored (should be in `UnsupportedProperties`, not sur why it's not…)

Reply:
The depends_on option is ignored when deploying a stack in swarm mode with a version 3 Compose file. 

Good catch.